src/components/Header/NetworkSelector.tsx

```diff
@@ -195,7 +195,7 @@ function Row({
     return null
   }
   const active = chainId === targetChain
-  const { helpCenterUrl, explorer, bridge, label, logoUrl } = CHAIN_INFO[targetChain]
+  const { helpCenterUrl, explorer, bridge, label, logoUrl } = getChainInfo(targetChain)
```

Comment:
this could fail, right? since it might be undefined. i'm surprised we're allowed to destructure a possibly undefined object.

Reply:
or does this work because targetChain is a `SupportedChainId`?